google/devtools/clouddebugger/clouddebugger.yaml

```diff
@@ -0,0 +1,38 @@
+type: google.api.Service
+
+config_version: 2 # (assumed)
```

Comment:
Assumed by the script in https://github.com/googleapis/googleapis/pull/146 when it inputs a service config without a version set.